app/timetables/models.py

```diff
@@ -96,3 +96,20 @@ def save(self, *args, **kwargs):
 
     def __str__(self):
         return self.name
+
+
+class Dish(TimestampMixin):
+    """
+    Model representing Dish.
+
+    A dish represents the actual food served as a given
+    course under an option of a meal on a cycle day in a timetable.
+    E.g, Coconut rice garnished with fish stew and chicken or just
+    Ice-cream.
+    """
+
+    name = models.CharField(max_length=255, unique=True)
+    description = models.CharField(max_length=255)
```

Comment:
Adding description field was a separate ticket for https://github.com/teamtaverna/core/issues/51. Since you have updated it here, I will reference the issue in this ticket so you can finish both alongside. Note that the description field is an optional one in the model, so it can be left blank.